www/src/components/shell/onboarding/OnboardingLogo.tsx

```diff
@@ -1,14 +1,11 @@
-import {
-  Div,
-  Flex,
-  Hr,
-  Img,
-} from 'honorable'
+import { Div, Flex, Img } from 'honorable'
+
+// TODO: Export to the Design System
```

Comment:
I'd recommend moving all TODOs from code to Linear before merge. It will be easier to keep track of them.

Reply:
Done: https://linear.app/pluralsh/issue/ENG-1148/export-onboardinglogo-to-the-design-system